book/tutorials/photon_classifier.py

```diff
@@ -0,0 +1,178 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.2
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Machine Learning with ICESat-2 data
+#
+# A machine learning pipeline from point clouds to photon classifications.
+#
+# Reimplementation of
+# https://github.com/YoungHyunKoo/IS2_ML/blob/main/01_find_overlapped_data.ipynb
+
+# %% [markdown]
+# ```{admonition} Learning Objectives
+# By the end of this tutorial, you should be able to:
+# - Convert ICESat-2 point cloud data into an analysis/ML-ready format
+# - Recognize the different levels of complexity of ML approaches and the
+#   benefits/challenges of each
+# - Learn the potential of using ML for ICESat-2 photon classification
+# ```
+
+# %% [markdown]
+# ## Part 0: Setup
+
+# %%
+import geopandas as gpd
```

Comment:
Getting this error on the Netlify build:

```python-traceback
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 import geopandas as gpd
      2 import h5py
      3 import icepyx as ipx

ModuleNotFoundError: No module named 'geopandas'
```

Might need to either sync this repo's environment.yml file to match with CryoCloud [hub-image](https://github.com/CryoInTheCloud/hub-image), or push the pre-rendered notebook cells.

Reply:
Resolved in https://github.com/ICESAT-2HackWeek/website-2024/pull/18